healthvault-ios-sdk/Classes/Controllers/MHVBrowserAuthBroker.m

```diff
@@ -91,85 +94,84 @@ - (void)showAuthenticationControllerFromViewController:(UIViewController *)viewC
             vc = [[[UIApplication sharedApplication] keyWindow] rootViewController];
         }
         
-        UINavigationController *navigationController = [[UINavigationController alloc] initWithRootViewController:self];
+        if (!self.browserController)
```

Comment:
I actually think we should create a new browser controller every time. I'm noticing that the previous web view displays old content while navigating to the new page. I see that you are hiding the web view by setting the alpha to zero, but there might still be some leftover content from a previous session.